contents/blog/data-management-feature.md

```diff
@@ -0,0 +1,83 @@
+---
+date: 2022-03-23
+title: You can now organize your PostHog data
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+author: ["alex-kim"]
+categories: ["Product updates"]
+---
+
+As PostHog grows, the data that users are sending to PostHog continues to grow into the billions.
+
+Data explosion is real, and we've been working hard to deliver a solution to your data management woes. Today we're releasing our Data Management feature, which introduces new ways to help you organize events and properties being sent to PH. 
+
+This is the first step in a continuous effort to help make organizing your data easier:
```

Comment:
I would consider targeting this based on use cases & problems rather than functionality (we can list functionality in the feature page). Example: engineers/PMs sharing context so that marketing users can figure out their attributions.